LoopUI/Views/NotificationsCriticalAlertPermissionsView.swift

```diff
@@ -21,25 +21,37 @@ public struct NotificationsCriticalAlertPermissionsView: View, HorizontalSizeCla
     }
     
     public var body: some View {
-        NavigationView {
-            List {
-                manageNotificationsSection
-                manageCriticalAlertsSection
-                notificationAndCriticalAlertPermissionSupportSection
-            }
-            .listStyle(GroupedListStyle())
-            .navigationBarTitle(Text(LocalizedString("Alert Permissions", comment: "Notification & Critical Alert Permissions screen title")))
-            .navigationBarBackButtonHidden(false)
-            .navigationBarHidden(false)
-            .navigationBarItems(leading: dismissButton)
-            .environment(\.horizontalSizeClass, horizontalOverride)
+        if backButtonText == "" {
```

Comment:
Is using an empty string the clearest signifier here?
In a similar situation, I recommended to Anna to use something like

```swift
enum PresentationMode {
    case modal, flow
}
```

to make the choice of branch explicit.

Reply:
This is all temporary, until the new `SettingsView` is done. Eventually this check will completely go away.  Is it an issue?

Reply:
Ah if it's just for the transition, no problem 👍

Reply:
Actually, sounds ok to me...nevermind, I'll tackle it.

Reply:
Incidentally, @mpangburn, I noticed that `NotificationsCriticalAlertPermissionsView` does a little "hop" after transition... do you know what's causing that?  (see https://drive.google.com/file/d/1u85U00NXj2CqeiojyoPun_D39rz9ryqK/view?usp=sharing)

Reply:
Hmm, my eyes aren't quite picking up the hop, is it the whole view? Wouldn't surprise me if it were a SwiftUI quirk, navigation can be finnicky in SwiftUI

Reply:
Hm. Weird.  You are right it doesn't show up on that screen cap.  Try that link again... (https://drive.google.com/file/d/1u85U00NXj2CqeiojyoPun_D39rz9ryqK/view?usp=sharing)

Reply:
Looks like a bug @novalegra asked me about this morning!
I'm inclined to believe it's a SwiftUI+navigation issue. I know Pete and Nate found navigation using SwiftUI to be very buggy.

Reply:
I'm still not seeing the hop - what's happening?